eng/verify-nupkgs.ps1

```diff
@@ -17,18 +17,18 @@ Add-Type -AssemblyName System.IO.Compression.FileSystem
 function Verify-Nuget-Packages {
     Write-Host "Starting Verify-Nuget-Packages."
     $expectedNumOfFiles = @{
```

Comment:
For all packages using MIT license, reduce count by 1 as license file is no longer present.